Broadband/broadband_tracts.R

```diff
@@ -0,0 +1,69 @@
+# Get Raw Data and save to 'source' folder
+# Author: Eleanor Noble
+# Date: 6/19/2019
+# SET UP ==============================================
+pkgs <- c("tidyverse", "reshape2", "writexl", "httr","skimr", "janitor","stringr")
+
+check <- sapply(pkgs, require, warn.conflicts = TRUE, character.only = TRUE)
+if (any(!check)) {
+  pkgs.missing <- pkgs[!check]
+  install.packages(pkgs.missing)
+  check <- sapply(pkgs.missing, require, warn.conflicts = TRUE, character.only = TRUE)
+}
+
+# TRANSFORM ============================================
+# Patent Complexity ---------------------------------------------------
+# Broadband ---------------------------------------------------
+tract_broadband <- readxl::read_xlsx("V:/Infrastructure/2 Long Form Projects/Broadband/Final Layout/Masterfile_Final.xlsx")%>%
+  janitor::clean_names()%>%
+  mutate(tract_code = as.character(str_sub(tract,-6,-1)))
+
+#delete or rename columns
+tract_broadband$v1<-NULL
+tract_broadband$tractonly_fips<-NULL
+tract_broadband$geotype<-NULL
+tract_broadband$place<-NULL
+tract_broadband$stplfips<-NULL
+
+names(tract_broadband)[names(tract_broadband) == 'tract'] <- 'stcotract_code'
```

Comment:
Use [dplyr::select](https://dplyr.tidyverse.org/reference/select.html) to delete or rename columns. The code is more readable and intuitive.